crates/sui-config/src/genesis.rs

```diff
@@ -940,6 +939,10 @@ fn build_unsigned_genesis_data(
             SuiFramework::as_modules(),
             SuiFramework::transitive_dependencies(),
         ),
+        (
+            SuiSystem::as_modules(),
+            SuiSystem::transitive_dependencies(),
+        ),
```

Comment:
You'll need to add this as a parameter in the call to `create_genesis_context` above as well on like 923.

Reply:
Addressed this in the latest commit